django/utils/crypto.py

```diff
@@ -9,10 +9,15 @@
 from django.utils.encoding import force_bytes
 
 
-def salted_hmac(key_salt, value, secret=None):
+class InvalidAlgorithm(AttributeError):
+    pass
+
+
+def salted_hmac(key_salt, value, secret=None, algo=None):
```

Comment:
Prefer `algorithm` to `algo`. Also, make it keyword-only? I think that the default can be specified here too, rather than the `algo or 'sha1'` thing below.

```python
def salted_hmac(key_salt, value, secret=None, *, algorithm='sha1'):
```